hypershift-operator/controllers/nodepool/nto.go

```diff
@@ -392,7 +430,15 @@ func getMirrorConfigForManifest(manifest []byte) (*MirrorConfig, error) {
 
 	switch cr.(type) {
 	case *mcfgv1.ContainerRuntimeConfig:
-		mirrorConfig = &MirrorConfig{Labels: map[string]string{ContainerRuntimeConfigConfigMapLabel: ""}}
+		mirrorConfig = &MirrorConfig{Labels: map[string]string{
+			ContainerRuntimeConfigConfigMapLabel: "true",
+			MirroredConfigLabel:                  "true",
```

Comment:
how was this working without the MirroredConfigLabel before?

Reply:
I did this change across all the relevant places in the code that are trying to list the ConfigMaps in order to keep is explicit and consistent.
On NTO side we're relying on the `ContainerRuntimeConfig` labels anyway.